### ISSUE TITLE
puppet-lint fails to validate a hash

### ISSUE THREAD

**Issue by Qsick** [rodjek/puppet-lint#632]
I'm receiving the following error for a hash I have in my puppet code:

```
ERROR: Syntax error (try running `puppet parser validate <file>`) on line 86

shell returned 1
```
Here is the hash it is complaining about:
```
  $config => {
      'action.destructive_requires_name'     => true,
      'bootstrap.mlockall'                   => true,
      'cluster.name'                         => $elastic_cluster_name,
      'discovery.zen.minimum_master_nodes'   => $discovery_zen_minimum_master_nodes,
      'discovery.zen.ping.multicast.enabled' => false,
      'discovery.zen.ping.unicast.hosts'     => $elasticsearch_nodes,
      'gateway.recover_after_nodes'          => 5,
      'index.number_of_shards'               => 10,
      'index.number_of_replicas'             => 2,
      'node.name'                            => $::hostname,
      'node.data'                            => true,
      'node.master'                          => true,
      'script.engine.groovy.inline.update'   => true,
      'network.host'                         => $::facts['networking']['ip'],
    },
```

The line causing the error is:
```
      'discovery.zen.minimum_master_nodes'   => $discovery_zen_minimum_master_nodes,
```

If I comment out that line it goes further only to report a syntax error on the following line also:

```
      'gateway.recover_after_nodes'          => 5,
```

**Comment by rnelson0** [comment]
When I copy and paste your example into a file, it looks like there are invalid characters there. See http://imgur.com/a/CLcX7 for what it looks like when I edit with vi.

**Comment by rnelson0** [comment]
I've marked this as invalid due to the non-space characters.

**Comment by Qsick** [comment]
Yep, there were somehow invalid characters on those lines.  It's weird because I wasn't seeing them in vi with ':set list'

Thanks.
